zinc/src/sbt-test/source-dependencies/ext/A.scala

```diff
@@ -1,5 +0,0 @@
-import sun.net.spi.nameservice.dns.DNSNameService
```

Comment:
Can we instead move this to `pending`?

Reply:
If it passes in JDK 8 but fails on JDK 11, then we can't.

Reply:
sounds good